how-to/customize-workspace/public/manifest.fin.json

```diff
@@ -146,6 +146,106 @@
 				"newTabUrl": "http://localhost:8080/common/views/platform/new-tab/new-tab.html",
 				"newPageUrl": "http://localhost:8080/common/views/platform/new-tab/new-tab.html"
 			},
+			"globalMenu": [
+				{
+					"include": true,
+					"label": "Toggle Notification Center",
+					"data": {
+						"type": "Custom",
+						"action": {
+							"id": "notification-toggle"
+						}
+					},
+					"position": {
+						"type": "OpenStorefront",
+						"operation": "insertAfter"
+					}
+				},
+				{
+					"include": true,
+					"label": "Open Home",
+					"data": {
+						"type": "Custom",
+						"action": {
+							"id": "home-show"
+						}
+					},
+					"position": {
+						"type": "OpenStorefront",
+						"operation": "insertAfter"
+					}
+				},
+				{
+					"include": true,
+					"label": "Quit App",
+					"position": {
+						"type": "Quit",
+						"operation": "replaceLabel"
+					}
+				},
+				{
+					"include": true,
+					"label": "Log Out and Quit App",
+					"data": {
+						"type": "Custom",
+						"action": {
+							"id": "logout-and-quit"
+						}
+					},
+					"position": {
+						"type": "Quit",
+						"operation": "insertAfter"
+					},
+					"conditions": {
+						"isAuthenticationEnabled": true
+					}
+				}
+			],
+			"pageMenu": [
+				{
+					"include": true,
+					"label": "Move Page to new Window",
+					"data": {
+						"type": "Custom",
+						"action": {
+							"id": "move-page-to-new-window"
+						}
+					},
+					"position": {
+						"operation": "start"
+					},
+					"separator": "after"
+				}
+			],
+			"viewMenu": [
+				{
+					"include": true,
+					"label": "Move View(s) to new Window",
+					"data": {
+						"type": "Custom",
+						"action": {
+							"id": "move-view-to-new-window"
+						}
+					},
+					"position": {
+						"operation": "start"
+					},
+					"separator": "after"
+				},
+				{
+					"include": true,
+					"label": "Create App Definition",
+					"data": {
+						"type": "Custom",
+						"action": {
+							"id": "raise-create-app-definition-intent"
+						}

```

Comment:
I think we should have this action as an action module. Raising an intent for app definition is very specific and is for learning and won't fit in all cases.

Reply:
Not sure if the other custom actions need to be broken out as they are quite generic and can be excluded via meta data. This also means we have an example of loading a custom action that applies to menu options and not buttons.

Reply:
I have separated it out, it also means extending the helpers for actions a bit to pass additional data - otherwise the module ends up getting bloated with workspace module code.

Reply:
Cool. Could we move these new settings into the settings.json file in public which is used by second.manifest.fin.json so that it has the same options is had before?